front/app/containers/IdeasNewPage/index.tsx

```diff
@@ -417,7 +418,12 @@ export default withRouter((inputProps: InputProps & WithRouterProps) => {
     name: 'dynamic_idea_form',
   });
 
-  if (isDynamicIdeaFormEnabled) {
+  const project = useProject({ projectSlug: inputProps.params.slug });
+
+  if (
+    isDynamicIdeaFormEnabled ||
+    project?.attributes.participation_method === 'native_survey'
```

Comment:
About `project?`: in what circumstances could `project` be undefined or null?

Reply:
Looking at how the `useProject` hook is implemented, the project can be undefined or null. Before any effect is run to set the project, the initial value is undefined.

Reply:
ok